apps/src/templates/teacherNavigation/UnitCalendar.tsx

```diff
@@ -22,7 +22,7 @@ import styles from './teacher-navigation.module.scss';
 const WEEKLY_INSTRUCTIONAL_MINUTES_OPTIONS = [
   45, 90, 135, 180, 225, 270, 315, 360, 405, 450,
 ];
-export const WEEK_WIDTH = 585;
+export const WEEK_WIDTH = 800;
```

Comment:
The thing I don't know about here is does this work when it is a pop-up... this was hard to test because right now the calendar button isn't shown on the Unit landing page when the Unit is unassigned to a section... but I think you might have a ticket for that already...  so I am guessing that will be checked/debugged at that point.